src/main/lrsql/spec/statement.clj

```diff
@@ -78,6 +79,7 @@
 ;; - verb_iri:         STRING NOT NULL
 ;; - is_voided:        BOOLEAN NOT NULL DEFAULT FALSE
 ;; - payload:          JSON NOT NULL
+;; TODO: timestamp and stored insts
```

Comment:
You left a TODO here

Reply:
eep, good call.

Reply:
I see that you removed the TODO, but it's still missing the `stored` and `timestamp` fields

Reply:
no, it's not.

Reply:
oooh you mean the comments. oy.

Reply:
9b8ed91c0da4c20b6b76c950bd700952a4fb47ac

Reply:
`- stored: STORED NOT NULL` - don't you mean `TIMESTAMP NOT NULL`?

Reply:
hah, brain fart i knew that

Reply:
fixed